README.md

```diff
@@ -212,6 +213,47 @@ jobs:
 ...
 ```
 
+Single-workspace build, with large plan support:
+```yaml
+name: 'Terraform'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      TF_IN_AUTOMATION: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+          terraform_version: 0.15.0
+[...]
+      - name: Terraform Plan
+        id: plan
+        run: terraform plan -out workspace.plan 2>&1 | tee {{ github.workspace }}/terraform-plan.out
```

Comment:
should error messages really be in the plan output?

why is the outputfile written 2 times?

Reply:
> should error messages really be in the plan output?

This is how we pass the output currently stderr and stdout combined. Maybe this is wrong but I would change that in a separate commit/PR.
  
> why is the outputfile written 2 times?

What do you mean? The `-out` option does not write the output meant for human consumption. It is meant for `apply`.

```
  -out=path                  Write a plan file to the given path. This can be
                             used as input to the "apply" command.
```